src/pages/ProductList/ProductList.js

```diff
@@ -15,36 +15,39 @@ class ProductList extends Component {
   }
 
   componentDidMount() {
-    fetch(`${API.PRODUCTLIST}`)
+    fetch('/data/productList.json')
+      // fetch(`${API.PRODUCTLIST}`)
       .then(res => res.json())
       .then(data => {
         this.setState({
           productInfo: data.products,
           categoryInfo: data.category,
         });
-        console.log(data);
       });
   }
 
+  productHandler = arr => {
+    return arr.map(product => {
+      return (
+        <Product
+          key={product.id}
+          name={product.name}
+          price={product.price}
+          tag={product.tags}
+          img={product.thumbnail}
+        />
+      );
+    });
+  };
+
   render() {
     const { productInfo, categoryInfo } = this.state;
-
     return (
       <div className="productList">
         <ListHeader />
         <ListMenu />
         <div className="productContainer">
-          {productInfo.map(product => {
-            return (
-              <Product
-                key={product.id}
-                name={product.name}
-                price={product.price}
-                tag={product.tags}
-                img={product.thumbnail}
-              />
-            );
-          })}
+          {this.productHandler(productInfo)}
```

Comment:
- 다음과 같이 작성해주시면 훨씬 간결하고 이해하기 쉬운 코드가 됩니다!
```suggestion
          {productInfo.map(product => (
            <Product
              key={product.id}
              name={product.name}
              price={product.price}
              tag={product.tags}
              img={product.thumbnail}
            />
          ))}
```

Reply:
오 return 도 빠져도 되는군요!! 바로 반영 하겠습니다!!